README.md

```diff
@@ -1,53 +1,55 @@
+This project is designed with Kubevirt in mind and provides a declarative method of importing VM images into a Kuberenetes cluster. Through this behavior, cluster administrators can build an abstract registry of template images (referred to as "Golden Images" for their role as templates for image clones).
+
+For an in depth look at the system, see the [Design](/doc/design.md#design) documentation.
 
 ### Data Formatting
 
-While the importer has been designed to be relatively agnostic to file format, it is capable of performing certain functions that streamline its use with Kubevirt.  It will unpackage **gzip** and **tar** archived files automatically, as well as convert **qcow2** images into raw image files.  
+The importer is capable of performing certain functions that streamline its use with Kubevirt.  It will unpackage **gzip** and **tar** archived files automatically, as well as convert **qcow2** images into raw image files.
 
-## CDI Tire Kicking
+Expected file formats are:
 
-Deploying the controller pod is simple!
+- .tar
+- .gz
+- .img
+- .iso
+- .qcow2
 
-1. Secret(s) containing endpoint credentials. Not required if the endpoint(s) are public.
-1. Storage class(es) defining the backend storage provisioner(s).
-1. Controller pod via the Deployment template. Note that the controller pod spec needs to set an environment variable named OWN_NAMESPACE which can be done as:
-```
-...
-   imagePullPolicy: Always
-       env:
-         - name: OWN_NAMESPACE
-           valueFrom:
-             fieldRef:
-               fieldPath: metadata.namespace
-```
+## Deploying CDI
 
 ### Assumptions
 - A running Kubernetes cluster
-- A reachable object store
-- A file in the object store to be imported.
+- A storage class and provisioner.
+- An HTTP or S3 file server hosting VM images
+- A namespace acting as the image registry (`default` is fine for tire kicking)
+
+Deploying the CDI controller is straight forward.  Create the controller in the namespace where VM images are to be stored.
+
+`$ kubectl create -f https://raw.githubusercontent.com/kubevirt/containerized-data-importer/master/manifests/cdi-controller-deployment.yaml`
+
+
+To begin importing images, follow the PVC configuration guide below.
 
 ### Configuration
 
 Make copies of the [example manifests](./manifests/importer) to some local directory for editing.  There are several values required by the data importer pod that are provided by the configMap and secret.
```

Comment:
This link (https://github.com/copejon/containerized-data-importer/blob/165846bf77d49e79534af15bb59727ff2f7b8627/manifests/importer) goes to the importer pod's manifest. I think you want to point to the _manifests/_ dir.